runtime/src/main/resources/packaged/configuration.yml

```diff
@@ -1,35 +1,69 @@
+flux.Shard.Pair.Config: "{
```

Comment:
flux.Shard.Pair.Config: {
shardPairModelList: 
[
{...},
{...}
]
}

In the above model we can remove one level, we can model it as something like below

flux.Shard.Pair.Config: [
{...},
{...}
]